src/api/app/controllers/application_controller.rb

```diff
@@ -185,11 +250,8 @@ def extract_user
     if mode == :on
       extract_proxy_user
     else
-      @auth_method = :basic
-
-      extract_basic_auth_user
-
-      @http_user = User.find_with_credentials @login, @passwd if @login
+      extract_auth_user
+      @http_user = User.find_with_credentials @login, @passwd if @login && @passwd
```

Comment:
Why do we ask for a password here? With kerberos it should use the ticket, right?